### DISCUSSION TITLE
Error when make moose with phase field

### DISCUSSION THREAD

**Discussion by lizhi1008** [idaholab/moose#26226] [Q&A Getting Started]
Dear all,
This is my first time to use the phase field module in moose.
First, I run the stork.sh script to create an new application. Next, I modify the Makefile to enable the phase field module. Then, I make it but the following error occurred.
![QQ截图20231202152440](https://github.com/idaholab/moose/assets/152682506/54576904-f896-4a64-96c6-7f482437cff0)
It looks like the class EulerAngles is defined repeatedly.
I wonder how to fix it?
Thanks!

**Comment by GiudGiud** [comment]
Hello

The problem is probably that you compiled once without phase_field and now with phase_field.
So there's an old header around that gets picked up too early and defines symbols that get redefined later.

You need to clean the repository (`git commit` all your work then `make clobberall`. If that does not work use `git clean -xfd`)

Guillaume
